package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Low level internal utilities to be shared amongst Eleventy projects",
   "main": "index.js",
   "scripts": {
-    "test": "npx ava --verbose"
+    "test": "node --test"
   },
   "license": "MIT",
   "engines": {
-    "node": ">=12"
```

Comment:
I think this might be an issue. Is it possible to have different engine versions for dev and prod?

Reply:
(to be clear I think eleventy-utils is used in more places than core)

Reply:
If the dev engines require 18+ then we can’t test this package in environments where it needs wider compatibility. I think we’ll need to major version bump this with Node 18+ (which is fine) and opt-in to it with packages that are Node 18+.

I’m good with this.